.github/workflows/labeler.yaml

```
@@ -0,0 +1,46 @@
+permissions:
+  pull-requests: write
+  contents: read
+  issues: write
+
+jobs:
+  labeler:
+    runs-on: ubuntu-latest
+    name: Label the PR
+    steps:
+      - name: Add Labels for PR
+        uses: actions/labeler@v3
```

actionlint (1.7.7)

the runner of "actions/labeler@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)